apps/blog/docusaurus.config.ts

```diff
@@ -8,8 +8,8 @@ const config: Config = {
   title: "Blog",
   tagline: "Grida Blog",
   favicon: "img/favicon.ico",
-  url: "https://blog.grida.co",
-  baseUrl: "/blog",
+  url: "https://grida.co",
```

Comment:
I'm actually not sure if this is safe to do. I think we better not change this part as well.
Its blog.grida.co/blog -> pointing to grida.co/blog, so based on that, the url should remain blog.grida.co I believe, as its where its actually hosted.

Reply:
I saw it's the same pattern from `apps/docs/docusaurus.config.ts`, that's why I changed the url here
 
https://github.com/gridaco/grida/blob/03e31d9c43c5dab19d654815170cabfb25c87a8c/apps/docs/docusaurus.config.ts#L44-L45

Reply:
Hmm. ok let me give it a try